asmrun/spacetime.c

```diff
@@ -186,6 +188,10 @@ static void maybe_reopen_snapshot_channel(void)
 
 extern void caml_spacetime_automatic_save(void);
 
+#ifndef SIGINT
+#define SIGINT -1
```

Comment:
Are there any systems that don't define `SIGINT`? If you pass -1 to `caml_set_signal_action` it will report an error which you ignore. That's pretty bad style. You could simply put `#ifdef SIGINT` around the call to `caml_set_signal_action`.